src/get-started/timestamping-tutorial.md

```diff
@@ -465,36 +625,22 @@ structures:
   }
   ```
 
-- the `PublicApi` structure which enables us to implement public API endpoints
-  for our service:
-
-  ```rust
-  pub struct PublicApi;
-  ```
-
-Now we are ready to define the three endpoints required for our service:
+### API for Transactions
```

Comment:
Should we mention that `handle_timestamp`, `handle_timestamp_proof` and `wire` must be implemented within `PublicApi`?

Reply:
I guess it's redundant because in the code below we can see
```rust
.public_scope()
```
method.